lib/bigint.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const bigintToInt32 = bn => {
+  const b1 = Number(bn & BigInt(0xffffffff));
+  const b2 = Number((bn << BigInt(32)) & BigInt(0xffffffff));
+  return [b1, b2];
+};
+
+const bigintFromInt32 = ([b1, b2]) => {
+  const bn1 = BigInt(b1);
```

Comment:
It'll be better to change `bn1`, `bn2` to `b1`, `b2` so variables names will stay consistent across this file